frontend/src/hooks/useWebMedia.ts

```diff
@@ -0,0 +1,50 @@
+import { useState, useEffect } from 'react'
+
+export const audioOptions = { audio: true, video: false }
+export const videoOptions = { audio: true, video: true }
+export const photoOptions = { audio: true, video: false }
+
+export const useWebMediaRecorder = (options: any) => {
```

Comment:
Any reason why this could not be typed?

```suggestion
interface MediaOptions {
  audio: boolean
  video: boolean
}

export const audioOptions: MediaOptions = { audio: true, video: false }
export const videoOptions: MediaOptions = { audio: true, video: true }
export const photoOptions: MediaOptions = { audio: true, video: false }

export const useWebMediaRecorder = (options: MediaOptions) => {
```